stars/inject_stars.sh

```diff
@@ -6,3 +6,4 @@ set -ex
 yarn install
 STARS=$(node stars.js)
 sed -i "s/githubStars: .*/githubStars: $STARS/" ../src/_data/header2.yml
+sed -i "s/githubStars: .*/githubStars: $STARS/" ../src/_data/footer2.yml
```

Comment:
Is this correct? Maybe it's preferable to combine into a single command?

Reply:
this seems fine